tests/FileManipulation/ThrowsBlockAdditionTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace Psalm\Tests\FileManipulation;
+
+class ThrowsBlockAdditionTest extends FileManipulationTestCase
+{
+    /**
+     * @return array<string,array{string,string,string,string[],bool}>
+     */
+    public function providerValidCodeParse(): array
+    {
+        return [
+            'addThrowsAnnotationToFunction' => [
+                '<?php
+                    function foo(string $s): string {
+                        if("" === $s) {
+                            throw new \InvalidArgumentException();
+                        }
+                        return $s;
+                    }',
+                '<?php
+                    /**
+                     * @throws InvalidArgumentException
+                     */
+                    function foo(string $s): string {
+                        if("" === $s) {
+                            throw new \InvalidArgumentException();
+                        }
+                        return $s;
+                    }',
+                '7.4',
+                ['MissingThrowsDocblock'],
+                true,
+            ],
+            'addMultipleThrowsAnnotationToFunction' => [
+                '<?php
+                    function foo(string $s): string {
+                        if("" === $s) {
+                            throw new \InvalidArgumentException();
+                        }
+                        if("" === \trim($s)) {
+                            throw new \DomainException();
+                        }
+                        return $s;
+                    }',
+                '<?php
+                    /**
+                     * @throws InvalidArgumentException|DomainException
+                     */
+                    function foo(string $s): string {
+                        if("" === $s) {
+                            throw new \InvalidArgumentException();
+                        }
+                        if("" === \trim($s)) {
+                            throw new \DomainException();
+                        }
+                        return $s;
+                    }',
+                '7.4',
+                ['MissingThrowsDocblock'],
+                true,
+            ],
+            'preservesExistingThrowsAnnotationToFunction' => [
+                '<?php
+                    /**
+                     * @throws InvalidArgumentException|DomainException
+                     */
+                    function foo(string $s): string {
+                        if("" === $s) {
+                            throw new \Exception();
+                        }
+                        return $s;
+                    }',
+                '<?php
+                    /**
+                     * @throws InvalidArgumentException|DomainException
+                     * @throws Exception
```

Comment:
i preserve the existing throws annotation and add a new one based on inference

Reply:
So the fix will always add a new throws annotation which can have `Exception1|Exception2` syntax, but not modify existing one